src/Storageless/Http/SessionMiddleware.php

```diff
@@ -173,16 +174,14 @@ public static function fromAsymmetricKeyDefaults(
      * @throws \InvalidArgumentException
      * @throws \OutOfBoundsException
      */
-    public function __invoke(Request $request, Response $response, callable $out = null) : Response
+    public function process(Request $request, DelegateInterface $delegate) : Response
```

Comment:
I already had a chat about it with @weierophinney. It won't change, and that's better: no more breaking changes.